enterprise/backend/src/metabase_enterprise/embed/content_translation/api.clj

```diff
@@ -0,0 +1,37 @@
+(ns metabase-enterprise.embed.content-translation.api
```

Comment:
thoughts about putting this in `metabase-enterprise.content-translation.api.embed` instead? Not sure we want an entirely new separate module for embedding, especially since it seems to wholly depend on content translation?